pulp_rpm/app/serializers/repository.py

```diff
@@ -79,33 +79,18 @@ class RpmRepositorySerializer(RepositorySerializer):
         required=False,
         allow_null=True,
     )
-    gpgcheck = serializers.IntegerField(
-        max_value=1,
-        min_value=0,
-        default=0,
-        required=False,
-        help_text=_(
-            "An option specifying whether a client should perform "
-            "a GPG signature check on packages."
-        ),
-    )
-    repo_gpgcheck = serializers.IntegerField(
-        max_value=1,
-        min_value=0,
-        default=0,
-        required=False,
-        help_text=_(
-            "An option specifying whether a client should perform "
-            "a GPG signature check on the repodata."
-        ),
-    )
```

Comment:
This is an API breaking change, we probably need to convert it into a deprecation warning and disallow using it in conjunction with `repo_config`.

It's also exposed in pulp-cli, so we'll want to remove them from there.  And then after a couple releases we can drop them.

Reply:
True that it's an API change, however with the default behavior the .repo file is not generated anyway so I don't know what kind of big impact it would have.
Forgot about the CLI though.
I will follow your suggestion on adding deprecation warning